book.rb

```diff
@@ -0,0 +1,10 @@
+class Book
+  has_many :rentals
+
+  attr_accessor :title, :author
+
+  def initialize(title, author)
+    @title = title
+    @author = author
```

Comment:
- [ ] Kindly add a method for adding rentals should be present. you can check this lesson  [lesson](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship),  so to achieve the association here this method add rental it should be added here and in person class as I mentioned before as well so here the book itself should be sent as a parameter to create the new rental achieving the 'has-many' association.`💯✅✅🔥🚀